### PR TITLE
New - Add LightService 9.1.0

### PR DESCRIPTION
**Pull request by contactsamie**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib

Git repository (required):https://github.com/contactsamie/LightService
Official website (optional, not the repository):
NPM package url (optional): https://www.npmjs.com/package/lightservice
GitHub / Bitbucket popularity (required):
Count of stars:1
Count of watchers:1
Count of folks:1
NPM download stats (optional):
Downloads in the last day:18
Downloads in the last week:1064
Downloads in the last month:1695
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [ ] This lib was not found on cdnjs repo
- [ ] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [ ] Has valid tags for each versions (for git auto-update)
- [ ] Auto-update setup
- [ ] Auto-update target/source is valid.
- [ ] Auto-update filemap is correct.
# Git commit checklist
- [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [ ] The parent of the commit(s) in the PR is not old than 3 days.
- [ ] Pull request is sending from a non-master branch with meaningful name.
- [ ] Separate unrelated changes into different commits.
- [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [ ] Close corresponding issue in commit message
- [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by contactsamie** [comment]
I'll come back to the PR

File: ajax/libs/lightservice/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+   "version": "9.1.0",
+   "filename": "lightservice.min.js",
+   "name": "lightservice",
+   "homepage": "https://github.com/contactsamie/LightService",
```
Comment (kennynaoh) [inline ajax/libs/lightservice/package.json]: Hello @contactsamie , we will use git repo as default homepage. Please remove this field. Thanks!

**Comment by contactsamie** [comment]
Done : Removed home page

**Comment by PeterDaveHello** [comment]
Hello @contactsamie,

Since this PR is a little bit old, and also we do not accept PR from master branch anymore, would you like to send a new one base on the latest master branch? Many thanks!

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by contactsamie** [comment]
Hi @PeterDaveHello  Thanks for reaching out. I haven't done much on this as of yet. I'll update you

**Comment by PeterDaveHello** [comment]
@contactsamie Thanks.

**Comment by sufuf3** [comment]
Hi @contactsamie  Do you want to continue to work on this?

**Comment by sufuf3** [comment]
@PeterDaveHello There is no response from the author since 26 Apr.
And the lib doesn't have enough popularity.
And as your comment, https://github.com/cdnjs/cdnjs/pull/7057#issuecomment-272524766 said this PR need to be close and create another one by creating a new branch.

Should we help this lib to host in CDNJS?
Or just close this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
@contactsamie please send a new PR when you have time, thanks.

**Pull request closed without merging**